### ISSUE TITLE
fringe-indicator-mode has no effect

### ISSUE THREAD

**Issue by zoechi** [Alexander-Miller/treemacs#644]
I have fringe-indicator-mode enabled (see #641) but no indicator is shown.
Any idea why?
The whole fringe column that I have on other windows is not visible in the treemacs window.

**Comment by Alexander-Miller** [comment]
Do you use doom? Judging by the screenshots it gets rid of fringes in treemacs. There might be an option to get them back, if not you can do it yourself with 
```Elisp
(add-hook  'treemacs-mode-hook  (lambda () (set-window-fringes (selected-window) 40)))
```

**Comment by zoechi** [comment]
I have my custom Emacs config, but I use a Doom theme.
The code didn't have any noticeable effect.

**Comment by Alexander-Miller** [comment]
Did you kill the treemacs buffer and call it again? You can also try directly like this:
```Elisp
(with-selected-window (treemacs-get-local-window)
  (set-window-fringes nil 100))
```
If that doesn't bring back a 100px wide fringe then I don't know where to look, you'll need to debug your config for a probable cause.

**Comment by zoechi** [comment]
This is suddenly working. It seems to be related to theme settings.
I revived some settings related to `hl-line` and `fringe` from old commented out theme settings I used before I switched to doom theme and now the selected item shows a vertical white bar in the fringe column.
It seems to be related to the order the settings are invoked, because when I comment it out and restart and then invoke it manually, it doesn't affect the Treemacs window. Also not if I close and reopen. Only if I remove the comment and restart Emacs.

Thanks a lot for your support!
